packages/connect-examples/new-example/app/components/common/TemplateRegistryBoundary.tsx

```
@@ -0,0 +1,34 @@
+  children: React.ReactNode;
+  notFoundMessage?: string;
+  checkNotFound?: () => boolean;
+}
+
+export const TemplateRegistryBoundary: React.FC<TemplateRegistryBoundaryProps> = ({
+  children,
+  notFoundMessage = 'The requested resource could not be found.',
+  checkNotFound,
+}) => {
+  // Not Found 检查
+  if (checkNotFound && checkNotFound()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)